### PR TITLE
[SDK-1079][Docs] Adding MacOS support through Mac Catalyst

### PR DESCRIPTION
**Pull request by hokstuff**
# Pull Request/Issue Resolution
https://jira.braze.com/browse/SDK-1079

**Description of Change:**
Docs for adding MacOS support through Mac Catalyst

### Is this change associated with a Braze feature/product release?
- [x] Yes (__3.32.0__)
- [ ] No

Screen Shot 2021-01-25 at 3 35 03 PM

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+nav_title: Initial SDK Setup
+platform: MacOS
+page_order: 0
+---
+
+# Initial SDK Setup
+
+As of version [3.32.0][1], the Braze SDK supports MacOS for apps using [Mac Catalyst][2], which allows iPad apps to be built for Mac.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md:9]: 3.32.0 is a placeholder version number. TBD on which version we will release Catalyst support.

File: _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+{% alert note %}
+To build your app with Mac Catalyst, please reference <a href="https://developer.apple.com/documentation/uikit/mac_catalyst">Apple's documentation here</a>.
+{% endalert %}
+
+Once your app supports Catalyst, follow [these instructions to use Swift Package Manager][3] to import the Braze SDK into your app. Currently, the SDK does not support Mac Catalyst when using Cocoapods or Carthage.
+
+## Supported Features
+
+Braze supports push notifications and location features when running on Mac Catalyst. To integrate push in your Catalyst app, follow the [iOS SDK instructions here][4]. For enabling automatic location collection, follow the [setup instructions here][5].
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md:19]: Currently, I did not call out Rich Push here. I was thinking clients can reach out thru Support if it doesn't work for them

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md:19]: We should mention it imo

**Review by Bucimis — DISMISSED** [review]

**Review by lowip — APPROVED** [review]
LGTM, +1 on mentioning rich push

File: _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md (hunk 2 above)
Comment (chshapiro) [reply _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md:19]: +1

File: _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md [hunk 3]
```diff
@@ -52,8 +52,10 @@ Edit the scheme of the target including the Appboy package (_Product > Scheme >
...
 ```sh
-rm -rf "${TARGET_BUILD_DIR}/${PRODUCT_NAME}.app/Frameworks/libAppboyKitLibrary.a"
-rm -rf "${TARGET_BUILD_DIR}/${PRODUCT_NAME}.app/Plugins/libAppboyKitLibrary.a"
+# iOS
+bash "$BUILT_PRODUCTS_DIR/Appboy_iOS_SDK_AppboyKit.bundle/Appboy.bundle/appboy-spm-cleanup.sh"
+# macOS (if applicable)
+bash "$BUILT_PRODUCTS_DIR/Appboy_iOS_SDK_AppboyKit.bundle/Contents/Resources/Appboy.bundle/appboy-spm-cleanup.sh"
 ```
 
 ![Swift Package Manager: Menu 7][apple_initial_setup_7]
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md:61]: This screenshot (linked to the change for `assets/img/ios/spm/image6.png` below) now reflects the updated script ^
Screen Shot 2021-01-29 at 2 19 13 PM

File: _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md:9]: I would note up front here that support is available through SPM only

**Review by Bucimis — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md (hunk 1 above)
Comment (hokstuff) [reply _docs/_developer_guide/platform_integration_guides/macOS/initial_sdk_setup.md]: Updated

**Comment by hokstuff** [comment]
@KellieHawks This is ready to merge now - The Mac Catalyst feature was just released :)

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**